MIGRATION.md

```diff
@@ -8,6 +8,10 @@ npx @sentry/migr8@latest
 
 This will let you select which updates to run, and automatically update your code. Make sure to still review all code changes!
 
+## Deprecate `extractTraceParentData` export from `@sentry/core` & downstream packages
+
+Instead, import this directly from `@sentry/utils`.
```

Comment:
l: I'd probably add a hint here that `continueTrace` might be a suitable replacement

Reply:
makes sense, I'll wait till we merge that and reference this then!

Reply:
I've added the sentence below, WDYT?